Cargo.toml

```diff
@@ -78,3 +81,7 @@ targets = []
 [[example]]
 name = "simple_login"
 required-features = ["argon2"]
+
+[patch.crates-io]
+p256 = { git = "https://github.com/RustCrypto/elliptic-curves", rev = "136fed7944d53c0508b1a93cd97bdab46891bcf7" }
+voprf = { git = "https://github.com/facebook/voprf", rev = "5badeff8d2e904588313576c4993ceb282412e9c" }
```

Comment:
Unfortunately not, because that would require another release of `voprf` with facebook/voprf#101.

That would actually be great but alas there is no pre-release of `p256`/`p384`, basically making a pre-release of `voprf` pointless because it would be unusable.

Reply:
On that note, we can also just wait, I'm not in a hurry to get this merged.

Reply:
We are ready: https://github.com/facebook/voprf/pull/106.
Need another bump in `voprf` though.

Reply:
Should be ready now!